core/src/utils/wrapper_opaque.rs

```diff
@@ -83,7 +86,7 @@ impl<T> EncodeAsType for WrapperKeepOpaque<T> {
         use scale_encode::error::{Error, ErrorKind, Kind};
 
         let visitor = visitor::new(out, |_, _| {
-            // Check that the target shape lines up: any other shape but the composite is wrong.
+            // Check that the target shape lines up: any other shape but composite it wrong.
```

Comment:
Fix not quite right hehe :P

```suggestion
            // Check that the target shape lines up: any other shape but composite is wrong.
```